tests/test_addons_oauth.py

```diff
@@ -118,6 +119,12 @@ def test_revoke_auth(self, mock_decorator, mock_deauth):
             external_account=self.external_account,
             user=self.user
         )
+        self.user_settings.reload()
+        assert_equal(
+            self.user_settings.oauth_grants,
+            {self.project._id: {self.external_account._id: {}}}
```

Comment:
Is `external_account.id == {}` a function of it being a mocked test? (The real question I'm asking is: is this right? I mean, I presume so, but it looks a little odd)